core/App/screens/Settings.tsx

```diff
@@ -61,6 +64,21 @@ const Settings: React.FC<SettingsProps> = ({ navigation }) => {
   return (
     <SafeAreaScrollView>
       <View style={styles.container}>
+        <Text style={styles.groupHeader}>User Settings</Text>
+        <View style={styles.rowGroup}>
+          {/* <Text style={styles.subHeader}>Name</Text> */}
```

Comment:
Is this supposed to be commented out?

Reply:
I left it commented out so it would follow bifold conventions for now but could easily be patched to the holdr design